tools/accuracy_checker/accuracy_checker/adapters/text_recognition.py

```diff
@@ -15,6 +15,7 @@
 """
 
 from collections import defaultdict
+from scipy.special import softmax
```

Comment:
please avoid usage scipy if it can be not used.
this lib can not be installed on some supported OSes

Reply:
Used softmax from ..utils. Modified it (added axis arg and keepdims=True), models which are dependent on this function should be tested, possibly I've broke something. :)